spec/system/admin/deactivations_spec.rb

```diff
@@ -27,7 +27,9 @@
 
       fill_in 'Email', with: other_admin.email
       fill_in 'Password', with: other_admin.password
-      click_button 'Sign in'
+      within 'form' do
```

Comment:
Appeasing Rubocop caused these to be ambiguous. This was the easiest fix; using test-ids is probably better but I wanted to avoid touching templates for this pass

Reply:
Strange they've added this rule when it can lead to issues like this. I think `click_button` and `click_link` communicates the intent way better and avoids these unambiguous matches. Standardrb may be in our future lol

Reply:
On closer inspection this is a bit of a stitch up 😆 they've deprecrated this rule because of this exact reason.

[Rule that was flagged](https://docs.rubocop.org/rubocop-capybara/cops_capybara.html#capybaraclicklinkorbuttonstyle)

[Replaced by](https://docs.rubocop.org/rubocop-capybara/cops_capybara.html)

Unfortunately the change hasn't been released yet and the rule still screams.

I'll disable the rule, revert the changes and add a todo to remove it once the library gets updated

Reply:
Nice find! stitch up indeed 😆 
If its easier to keep those changes in, we can always change them back when the new release eventually appears.